apps/data-feeds-config-generator/src/data-services/fetchers/exchanges/bitfinex/bitfinex.ts

```diff
@@ -14,12 +16,23 @@ export class BitfinexAssetsFetcher
 {
   async fetchAssets(): Promise<AssetInfo<BitfinexAssetInfo>[]> {
     const assets = await fetchBitfinexInfo();
+    const prices = await fetchBitfinexPricesInfo(assets[0]);
     return assets[0].map(asset => {
+      const tickerAsset = `t${asset}`;
       const pair = splitPair(asset);
+      let price = prices.find(p => p[0] === tickerAsset);
+      if (!price) {
+        console.warn(`[Bitfinex] Price not found for symbol: ${tickerAsset}`);
+        price = [tickerAsset, '', '', '', '', '', '', 0];
+      }
       return {
-        ...pair,
+        pair: {
+          base: pair.pair.base,
+          quote: pair.pair.quote,
+        },
         data: {
-          symbol: asset,
+          symbol: tickerAsset,
```

Comment:
This will help us remove this https://github.com/blocksense-network/blocksense/blob/16419db7f150bc105e8f53c753f289a0e77543b1/apps/oracles/crypto-price-feeds/src/exchanges/bitfinex.rs#L64  in the future.